src/main/kotlin/org/malachite/estella/services/InterviewService.kt

```diff
@@ -42,18 +42,41 @@ class InterviewService(
     fun getLastInterviewIdFromApplicationId(applicationId: Int): PayloadUUID =
         getLastInterviewFromApplicationId(applicationId).getId()
 
-    fun getLastInterviewFromApplicationId(applicationId: Int): Interview =
+    fun getLastInterviewFromApplicationId(applicationId: Int, withPossibleHosts: Boolean = false): InterviewWithPossibleHosts =
         withExceptionThrower {
             getAllByApplicationId(applicationId).sortedBy { it.dateTime }.sortedWith { a, b ->
                 when {
                     a.dateTime == null -> -1
                     b.dateTime == null -> 1
                     else -> a.dateTime.compareTo(b.dateTime)
                 }
-
             }.first()
+        }.let { interview ->
+            if (withPossibleHosts) {
+                val possibleHosts = interview.applicationStage.application.applicationStages.first().stage.id.let {
```

Comment:
Pierdoła trochę za długie i bym chociaż połowę przeniósł do nowej linii lub więcej

Reply:
zobaczymy, co ctrl+alt+l powie

Reply:
Powiedziało, że ok, więc bym zostawił - pisali to mądrzejsi ludzie ode mnie, plus mieści się na jednym ekranie